templates/publisher/listing.html

```diff
@@ -140,6 +140,93 @@
           </div>
         </div>
 
+        <div class="row">
+          <div class="col-2">
+            <label>Category:</label>
+          </div>
+          <div class="col-4">
+            <input type="hidden" name="categories"
```

Comment:
Can't it be handled in the `state` directly? Do we need this additional hidden input for categories data?
We are pushing exactly the same data into state via JS couple of lines below in the template.

Reply:
:+1: good point https://github.com/canonical-websites/snapcraft.io/issues/1483